scripts/travis/maven_release.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -e
+
+releaseVersion=$1
+developmentVersion=$2
+scm_path=$(mvn help:evaluate -Dexpression=project.scm.url -q -DforceStdout)
+
+# Use full history for release
+git checkout -B "${TRAVIS_BRANCH}"
+# Add email to link commits to user
+git config user.email "${GIT_EMAIL}"
+
+if [ -z ${releaseVersion} ] || [ -z ${developmentVersion} ]; 
+    then echo "Please provide a Release and Development verison in the format <acs-version>-<additional-info> (6.3.0-EA or 6.3.0-SNAPSHOT)"
+         exit -1
+    # TODO: Set up continuous release. As of REPO-4735 the following is not required if release stage is manual
+    # mvn --batch-mode \
```

Comment:
Remove todo unless we are going to do it

Reply:
Removed